Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -777,14 +779,18 @@ namespace OpenInfraPlatform {
 						if (!convert3DPointsTo2D(boundID, plane, loopVertices2D, loopVertices3D, faceLoopReversed)) 
 						{
 							conversionFailed = true;
-							throw oip::UnhandledException(face);
+							std::stringstream ExceptionMessage;
+							ExceptionMessage << "#" << face->getId() << "= IfcFace: loop could not be projected";
+							throw oip::InconsistentGeometryException(face, ExceptionMessage.str().c_str());
 							continue;

```

Comment:
```suggestion
							throw oip::InconsistentGeometryException(face, "loop could not be projected");
```
The `continue` would never ever get executed, because the `throw` occurs. This is what was meant by the comment.

Reply:
Resolved:  d142297